simplified-ui-settings/src/main/java/org/nypl/simplified/ui/settings/SettingsFragmentAccount.kt

```diff
@@ -687,6 +694,15 @@ class SettingsFragmentAccount : Fragment() {
     }
   }
 
+  /**
+   * Hides or show sign up options if is user in accessing the NYPL
+   */
+  private fun hideCardCreatorForNonNYPL() {
+    if(this.account.provider.displayName != cardCreatorLibrary) {
```

Comment:
This should be:
```
if (this.account.provider.cardCreatorURI != null) { ...
```
That'll stop the code from breaking if anyone else introduces a card creator in their library.